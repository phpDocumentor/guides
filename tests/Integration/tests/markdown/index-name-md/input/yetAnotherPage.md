# Yet Another Page

Lorem Ipsum Dolor.
