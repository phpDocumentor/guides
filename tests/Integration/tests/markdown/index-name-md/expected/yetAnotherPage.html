<!-- content start -->

<div class="section" id="yet-another-page">
            <h1>Yet Another Page</h1>

    <p>Lorem Ipsum Dolor.</p>

    </div>
<!-- content end -->
